shared/chat/conversation/giphy/index.desktop.js

```diff
@@ -41,7 +39,7 @@ const GiphySearch = (props: Props) => {
           })}
         </Kb.Box2>
       </Kb.ScrollView>
-      <Kb.Image src={poweredByImg} style={styles.poweredBy} />
+      <Kb.Icon type="icon-powered-by-giphy-120-26" style={styles.poweredBy} />
```

Comment:
I don't think you need to specify a height in `styles.powered` style, plus it's 26, not 25.